client/my-sites/email/email-provider-features/style.scss

```diff
@@ -26,6 +26,18 @@
 	}
 }
 
+.email-provider-features__logos {
+	display: flex;
+	flex-wrap: wrap;
+	margin-left: 21px;
+	margin-top: 20px;
+
+	img {
+		margin-left: 5px;
```

Comment:
> Should this only apply to the second and later items? e.g. :not( :first-child )

That's what I've used initially but then this would not play well when items are wrapped.

>  I am thinking it may be better to use column-gap: 5px; on the parent item to put 5px between the images.

I didn't know we could use that rule with flexbox. I'll update the code.